### DISCUSSION TITLE
Exclude the particular path in  vercel routes

### DISCUSSION THREAD

**Discussion by pugazhisha** [vercel/community#3418] [Help]
### Summary

Hai Team,

We are looking some functionality in vercel app like exclude the specific or particular  path for specific domain.

### Example

I have domain like text1.com. I want server only assets like images, js, css for this domain its under /_next/ folder . So that  i need to write route like  / path is giving 404 or 403 expect /_next/*. Its possible to  archive this requirement in vercel. if possible could you please share the documents. 

### Steps to Reproduce

For Example:

I have domain like text1.com. I want server only assets like images, js, css for this domain its under /_next/ folder . So that  i need to write route like  / path is giving 404 or 403 expect /_next/*. Its possible to  archive this requirement in vercel. if possible could you please share the documents.

**Comment by amyegan** [comment]
It sounds like you want to customize the appearance of the URL. You can do this with the Next.js by configuring rewrites in `next.config.js`. You can learn more about it at https://nextjs.org/docs/api-reference/next.config.js/rewrites
